packages/compass-app-stores/src/stores/instance-store.js

```diff
@@ -243,6 +243,32 @@ store.onActivated = (appRegistry) => {
     store.refreshInstance(appRegistry);
   });
 
+  // Event emitted when the Databases grid needs to be refreshed
+  // We additionally refresh the list of collections as well
+  // since there is the side navigation which could be in expanded mode
+  appRegistry.on('refresh-databases', async() => {
```

Comment:
Should the refresh button for databases in the sidebar emit the same event instead of generic refresh-data?